app/controllers/questions/asset_sale_income_controller.rb

```diff
@@ -5,5 +5,11 @@ class AssetSaleIncomeController < QuestionsController
     def self.show?(intake)
       intake.sold_assets_yes?
     end
+
```

Comment:
A quick overview of all of these changes is that previously we were setting the "method_name" in the view as a content_for. However, I want to use this method_name to determine whether there is an unsure method when the method is an enu and we're loading into the yes/no question layout. Moving it into the controller as a helper method allows us to continue using it in the view but also have it available at the controller level.